fabtests/pytest/common.py

```diff
@@ -201,3 +201,36 @@ def run(self):
         for i in range(numclient):
             check_returncode(client_process_list[i].returncode, strict)
 
+def has_cuda(ip):
```

Comment:
commit message doesn't talk about the has_cuda function

Reply:
This commit has been removed for future